bindsnet/network/__init__.py

```diff
@@ -241,17 +245,19 @@ def run(self, inpts: Dict[str, torch.Tensor], time: int, **kwargs) -> None:
         
         # Simulate network activity for `time` timesteps.
         for t in range(timesteps):
-            # Update each layer of nodes.
+
+            # iterate on each layer
             for l in self.layers:
+                # Force neurons to certain voltage.
+                clamp = clamps.get(l, None)
+                if clamp is not None:
+                    self.layers[l].v = torch.where(clamp[t,:], clamps_v[l][t,:], self.layers[l].v)
```

Comment:
Should be `self.layers[l].v = torch.where(clamp, clamps_v[l], self.layers[l].v`; the implementation of `clamps` (thus far) assumes there is no time dimension.

Reply:
That right, now clamp have time dimension

Reply:
Why? The input is passed in, timestep by timestep.

Also, this breaks some of the examples.